packages/block-editor/src/components/media-placeholder/index.native.js

```diff
@@ -15,10 +15,10 @@ import { MediaUpload, MEDIA_TYPE_IMAGE, MEDIA_TYPE_VIDEO } from '@wordpress/bloc
 import styles from './styles.scss';
 
 function MediaPlaceholder( props ) {
-	const { mediaType, labels = {}, icon, onSelectURL } = props;
+	const { allowedTypes = [], labels = {}, icon, onSelect } = props;
 
-	const isImage = MEDIA_TYPE_IMAGE === mediaType;
-	const isVideo = MEDIA_TYPE_VIDEO === mediaType;
+	const isImage = MEDIA_TYPE_IMAGE === allowedTypes[ 0 ];
```

Comment:
Shouldn’t this be `allowedTypes.includes( MEDIA_TYPE_IMAGE )`?

Reply:
Resolved